bbb-learning-dashboard/src/components/PluginsTable.jsx

```diff
@@ -0,0 +1,161 @@
+            viewBox="0 0 24 24"
+            stroke="currentColor"
+          >
+            <path
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeWidth="2"
+              d="M9 5H7a2 2 0 00-2 2v12a2 2 0 002 2h10a2 2 0 002-2V7a2 2 0 00-2-2h-2M9 5a2 2 0 002 2h2a2 2 0 002-2M9 5a2 2 0 012-2h2a2 2 0 012 2m-3 7h3m-3 4h3m-6-4h.01M9 16h.01"
+            />
+          </svg>
+        </div>
+        <p className="text-lg font-semibold text-gray-700">
+          <FormattedMessage
+            id="app.learningDashboard.pluginsTable.noPluginsCreatedHeading"
+            defaultMessage="No plugin yet for this meeting"
+          />
+        </p>
+        <p className="mb-2 text-sm font-medium text-gray-600">
+          <FormattedMessage
+            id="app.learningDashboard.pluginsTable.noPluginsCreatedMessage"
+            defaultMessage="Once a plugin stores its data into learning-analytics-dashboard, it will appear in this list."
+          />
+        </p>
+      </div>
+    );
+  }
+
+  const commonUserProps = {
+    field: 'User',
+    headerName: intl.formatMessage({ id: 'app.learningDashboard.pluginsTable.userLabel', defaultMessage: 'User' }),
+    flex: 1,
+    sortable: true,
+  };
+
+  const commonCountProps = {
+    field: 'count',
+    headerName: intl.formatMessage({ id: 'app.learningDashboard.pluginsTable.answerTotal', defaultMessage: 'Total' }),
+    flex: 1,
+    sortable: true,
+  };
+
+  const gridCols = [
+    {
+      ...commonUserProps,
+      valueGetter: (params) => params?.row?.User?.name,
+      renderCell: (params) => (
+        <>
+          <div className="relative hidden w-8 h-8 rounded-full md:block">
+            <UserAvatar user={params?.row?.User} />
+          </div>
+          <div className="mx-2 font-semibold text-gray-700">{params?.value}</div>
+        </>
+      ),
+    },
+  ];
+
+  gridCols.push({
+    ...commonCountProps,
+    valueGetter: (params) => Object.keys(params?.row?.User?.pluginAnalytics)?.length || 0,
+    renderCell: (params) => params?.value,
+  });
+
+  pluginAnalyticsTitle.map((pluginAnalytics) => {
+    const commonColProps = {
+      field: pluginAnalytics,
+      headerName: pluginAnalytics,
+      flex: 1,
+    };
+    gridCols.push({
+      ...commonColProps,
+      valueGetter: (
+        params,
+      ) => params?.row?.[pluginAnalytics],
+      renderCell: (params) => params?.value,
+    });
+    return pluginAnalytics;
+  });
+
+  const gridRows = [];
+  Object.values(allUsers).map((u, i) => {
+    if (u?.isModerator && Object.keys(u?.pluginAnalytics)?.length === 0) return u;
+    gridRows.push({
+      id: i + 1,
+      User: u,
+      // This is going to be of the form:
+      // [learningAnalyticsDashboardColumnTitle]: learningAnalyticsDashboardValue, for each entry
+      ...u.pluginAnalytics.reduce((acc, curr) => {
+        const {
+          learningAnalyticsDashboardColumnTitle,
+          learningAnalyticsDashboardValue,
```

Comment:
I think you could name this columns differently once it is already inside of a learningDashboard object, it is not necessary to contain it in the name: `columnName` and `value` would be alright.